From b47c85c2c6eb35ef2c5c631bc6fddf4b432a1ec8 Mon Sep 17 00:00:00 2001
From: Purdea Andrei <andrei@purdea.ro>
Date: Sun, 21 Jun 2020 05:17:39 +0300
Subject: [PATCH] Add the .ino extension to the list of extensions recognized
 by the c++ syntax highlighter.

.ino is used by the Arduino IDE, and a number of other IDEs, and it's effectively the same
thing as a c++ source file, but with some added magic, such as auto-generated function prototypes.
---
 misc/syntax/Syntax.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/misc/syntax/Syntax.in b/misc/syntax/Syntax.in
index 724447bef..397bec473 100644
--- a/misc/syntax/Syntax.in
+++ b/misc/syntax/Syntax.in
@@ -97,7 +97,7 @@ include texinfo.syntax
 file ..\*\\.c$ C\sProgram
 include c.syntax
 
-file ..\*\\.([hC]|(?i:cxx|cc|cpp|hpp|hxx|hh)|[Hh]\\.in)$ C/C\+\+\sProgram
+file ..\*\\.([hC]|(?i:cxx|cc|cpp|hpp|hxx|hh|ino)|[Hh]\\.in)$ C/C\+\+\sProgram
 include cxx.syntax
 
 file ..\*\\.d$ D\sProgram
-- 
2.20.1

